docs/application/dotnet/guides/text-input/input-method.md

```diff
@@ -86,6 +86,41 @@ To start the IME application life-cycle:
     }
     ```
 
+## Making a Floating Keyboard
+
+To make the floating keybaord to allow the user to move the keyboard:
+
+1.  Call the `SetFloatingMode()` method.
+
+    ```
+    public static void SetFloatingMode()
+    {
+        InputMethodEditor.SetFloatingMode(true);
+    ```
+
+2.  Create a container of PanGestureRecognizer to detect a dragging event.
+
+    ```
+    var panGesture = new PanGestureRecognizer ();
+    panGesture.PanUpdated += OnPanUpdated;
+    GestureRecognizers.Add (panGesture);
+    ```
+
+3.  Call the SetFloatingDragStart() and SetFloatingDragEnd() method when a drag event is received.
```

Comment:
Please replace period with colon.

Please add code tag for `SetFloatingDragStart()`  and `SetFloatingDragEnd()`